package.json

```diff
@@ -35,20 +35,23 @@
     "xhr2": "0.0.7"
   },
   "devDependencies": {
-    "eslint-config-fxa": "1.8.1",
-    "grunt": "0.4.2",
-    "grunt-build-control": "git://github.com/robwierzbowski/grunt-build-control#274952",
-    "grunt-bump": "0.3.1",
-    "grunt-bytesize": "0.1.1",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-clean": "0.6.0",
-    "grunt-contrib-watch": "0.6.1",
-    "grunt-contrib-yuidoc": "0.9.0",
-    "grunt-conventional-changelog": "3.0.0",
-    "grunt-copyright": "0.2.0",
-    "grunt-eslint": "16.0.0",
-    "grunt-open": "0.2.2",
-    "grunt-webpack": "3.0.2",
+    "eslint": "5.14.1",
+    "eslint-config-fxa": "3.0.0",
+    "eslint-plugin-no-unsafe-innerhtml": "1.0.16",
+    "eslint-plugin-scanjs-rules": "0.2.1",
+    "grunt": "1.0.3",
+    "grunt-build-control": "0.7.1",
+    "grunt-bump": "0.8.0",
+    "grunt-bytesize": "0.2.0",
+    "grunt-cli": "1.3.2",
+    "grunt-contrib-clean": "2.0.0",
+    "grunt-contrib-watch": "1.1.0",
+    "grunt-contrib-yuidoc": "1.0.0",
+    "grunt-conventional-changelog": "6.1.0",
```

Comment:
this might break the changelog structure, or could be fine :)

Reply:
Ah yeah, it does. Didn't test that one, sorry. I'll put it back.

Reply:
...although I get the same error with `3.0.0` too:

```
No "conventionalChangelog" targets found.
```

Is it expected or is something wrong?